examples/shared_samples/logical_interconnect_group.rb

```diff
@@ -138,6 +138,16 @@
   puts "  #{r[:name]}"
 end
 
+if @client.api_version >= 600
+  # Gets a logical interconnect groups by scopeUris
+  query = {
+    scopeUris: '/rest/scopes/560a9d59-540a-4135-b8ad-ed97013ddfdc'
```

Comment:
we decided not to hard-code the scopeUris. Please have a look at other PR's

Reply:
Took the reference from one of the examples which is present in master. Should I leave it blank?